src/rocks-command/rocks/commands/report/host/vm/config/__init__.py

```diff
@@ -194,6 +194,9 @@ def getInterfaces(self, node):
 			returnxml.append("    <interface type='bridge'>")
 			returnxml.append("      <source bridge='%s'/>" % bridged_inter["device"])
 			returnxml.append("      <mac address='%s'/>" % bridged_inter["mac"])
+			returnxml.append("      <filterref filter='clean-traffic'>")
```

Comment:
http://libvirt.org/formatnwfilter.html#nwfexamples
clean-traffic	Prevent MAC, IP and ARP spoofing. This filter references several other filters as building blocks.

`[root@hpcdev-pub02 ~]# virsh nwfilter-dumpxml clean-traffic
<filter name='clean-traffic' chain='root'>
  <uuid>4fe52ec6-d09b-5fa3-de29-27ad99535191</uuid>
  <filterref filter='no-mac-spoofing'/>
  <filterref filter='no-ip-spoofing'/>
  <rule action='accept' direction='out' priority='-650'>
    <mac protocolid='ipv4'/>
  </rule>
  <filterref filter='allow-incoming-ipv4'/>
  <filterref filter='no-arp-spoofing'/>
  <rule action='accept' direction='inout' priority='-500'>
    <mac protocolid='arp'/>
  </rule>
  <filterref filter='no-other-l2-traffic'/>
  <filterref filter='qemu-announce-self'/>
</filter>`

Those other referenced filters are described in the same URL

I first ran a VM without 'clean-traffic' parameter, and was able to change the IP inside the guest and access the network from it. Then I enabled the 'clean-traffic' parameter and was only able to access the network when the IP is set to the value defined by its "IP" parameter

How it works is described here: http://libvirt.org/formatnwfilter.html#nwfwrite , I won't copy it here. It uses iptables and ip6tables and ebtables on the host, which should be a permanent rule

I can check whether the rules stay when iptables are restarted